hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -138,9 +143,11 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
         dropTableData(ops.io(), lastMetadata);
       }
 
+      LOG.info("Dropped table: {}", identifier);
       return true;
 
     } catch (NoSuchTableException | NoSuchObjectException e) {
+      LOG.debug("Skipping drop, table does not exist: {}", identifier, e);
```

Comment:
Should this be info?

I doubt that this is much help if you have to turn debug on and re-run because whether the table exists is external and modified by this method.

Reply:
Done